students/johnpharmd/lesson04/dict_lab.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Dictionaries01
+dict1 = {'name': 'Chris', 'city': 'Seattle', 'cake': 'Chocolate'}
+print(dict1)
+
+dict1.pop('cake')
+print(dict1)
+
+dict1['fruit'] = 'Mango'
+print(dict1)
+
+print(dict1.keys())
+print(dict1.values())
+print('"cake" is a key in the dictionary:', 'cake' in dict1)
+print('"Mango" is a value in the dictionary:', 'Mango' in dict1.values())
+print()
+
+# Dictionaries02
+dict2 = dict1.copy()
+print('dict1:', dict1)
+print('dict2:', dict2)
+
+for k, v in dict1.items():
+	dict2[k] = v.lower().count('t')
```

Comment:
your indentation is off (should be exactly 4 spaces).

most editors can be configured to have this set automatically.